### PR TITLE
[doc] Syntax highlighting

### PR DESCRIPTION
**Pull request by ara-ayvazyan**

### CODE REVIEW

File: doc/src/bond_java.md [hunk 1]
```diff
@@ -24,25 +24,30 @@ Basic example
 In Bond data schemas are defined using idl-like
 [syntax](compiler.html#idl-syntax):
 
+```
     namespace examples
```
Comment (chwarr) [inline doc/src/bond_java.md]: Using both \`\`\` and four spaces causes the code blocks all be indented an extra four spaces when the generated HTML is produced by Pandoc.

**Comment by ara-ayvazyan** [comment]
Yep, sending.

File: doc/src/compiler.md [hunk 2]
```diff
@@ -875,7 +875,7 @@ Other complex types are:
 
     ```javascript
     {
-      "type": "constant,
+      "type": constant,
```
Comment (chwarr) [inline doc/src/compiler.md]: It's actually `"type": "constant"` (both are strings).

Comment (ara-ayvazyan) [reply doc/src/compiler.md]: Ok.

**Pull request merged**